### PR TITLE
Add PureScript to the config

### PR DESCRIPTION
**Pull request by postsolar**

### CODE REVIEW

File: kak-lsp.toml [hunk 1]
```diff
@@ -267,6 +267,12 @@ filetypes = ["protobuf"]
 roots = [".git", ".hg"]
 command = "pls" # https://github.com/lasorda/protobuf-language-server
 
+[language_server.purescript]
+filetypes = ["purescript"]
+roots = [".git", "package.json", "spago.dhall", "spago.yaml"]
```
Comment (krobelus) [inline kak-lsp.toml:272]: roots are processed from left to right, so if there is any parent directory (including grandparent etc.) with a `.git` dir, that parent will be used as root. That's why we usually put .git/.hg last.
Do package.json/spago.dhall/spago.yaml have the same risk of false positives that `.git` has?

Comment (postsolar) [reply kak-lsp.toml:272]: Spago files are uniquely PureScript projects roots. `package.json` is not unique to PS so I guess it should come third?

**Comment by krobelus** [comment]
thanks! I made some cosmetic changes

**Comment by postsolar** [comment]
Thank you!

**Pull request closed without merging**